snare.py

```diff
@@ -319,6 +314,9 @@ def _run_cmd(cmd):
     parser.add_argument("--slurp-host", help="nsq logging host", default='slurp.mushmush.org')
     parser.add_argument("--slurp-auth", help="nsq logging auth", default='slurp')
     parser.add_argument("--config", help="snare config file", default='snare.cfg')
+
+    config = configparser.ConfigParser()
```

Comment:
Idea was to also read the config here. So either use the argument `--config` or the default path: `/opt/snare/snare.cfg`